config/config.php

```diff
@@ -74,6 +74,9 @@
 $CDASH_BERNARD_DRIVER = false;
 $CDASH_BERNARD_CONSUMERS_WHITELIST = false;
 
+// EXPERIMENTAL: Whether or not to use Bernard for submitting coverage jobs
+$CDASH_BERNARD_COVERAGE_SUBMISSION = false;
```

Comment:
What happens when `$CDASH_BERNARD_SUBMISSION` is true and `$CDASH_BERNARD_COVERAGE_SUBMISSION` is false?  I guess coverage builds fall back to one of our older processing pipelines based on whether or not `$CDASH_ASYNCHRONOUS_SUBMISSION` is set?